index.js

```diff
@@ -100,13 +108,34 @@ multiWorker.on('internalError', error =>
 multiWorker.on('multiWorkerAction', (verb, delay) =>
     winston.info(`*** checked for worker status: ${verb} (event loop delay: ${delay}ms)`));
 
+scheduler.on('start', () =>
+    winston.info('scheduler started'));
+scheduler.on('end', () =>
+    winston.info('scheduler ended'));
+scheduler.on('poll', () =>
+    winston.info('scheduler polling'));
+scheduler.on('master', state =>
+    winston.info(`scheduler became master ${state}`));
+scheduler.on('error', error =>
+    winston.info(`scheduler error >> ${error}`));
+scheduler.on('working_timestamp', timestamp =>
```

Comment:
Can you give us the link to the current doc for this version? Current README shows as `workingTimestamp`...

Reply:
There is no link in current README, their npm doc is still old, I put it as reference in the PR description. https://www.npmjs.com/package/node-resque

Reply:
Or you can clone this repo and do npm install and check inside the node_module. That's how I figure it out.. But couldn't find an easy link on their repo